apps/backoffice-v2/src/common/hooks/useSearchParamsByEntity/validation-schemas.ts

```diff
@@ -4,7 +4,7 @@ import { CaseStatus, CaseStatuses } from '../../enums';
 export const BaseSearchSchema = z.object({
   sortDir: z.enum(['asc', 'desc']).catch('desc'),
   pageSize: z.coerce.number().int().positive().catch(50),
-  page: z.coerce.number().int().positive().catch(1),
+  page: z.coerce.number().int().positive().catch(10),
```

Comment:
Why page 10 for a catch? What if there are no 10 pages?

Reply:
I forgot to remove that. Good catch